scripts/test-e2e-local-clean.js

```diff
@@ -44,24 +44,25 @@ if (isPackagerRunning() === 'running') {
 console.info('\n** Cleaning Gradle build artifacts **\n');
 exec('./gradlew clean');
 exec('rm -rf /tmp/maven-local');
+exec('rm -rf /tmp/maven-local.zip');
+exec('rm -rf /tmp/rntester.apk');
 
 // iOS
 console.info('\n** Nuking the derived data folder **\n');
 exec('rm -rf ~/Library/Developer/Xcode/DerivedData');
 
 console.info('\n** Removing the hermes-engine pod cache **\n');
 exec('rm -rf ~/Library/Caches/CocoaPods/Pods/External/hermes-engine');
+exec('rm -rf /tmp/hermes-ios-debug.tar.gz');
 
 // RNTester Pods
 console.info('\n** Removing the RNTester Pods **\n');
 exec('rm -rf packages/rn-tester/Pods');
 
-// I'm not sure we want to also remove the lock file
-// exec('rm -rf packages/rn-tester/Podfile.lock');
```

Comment:
why are you removing this?

Reply:
The lock file is committed to the repo, and there is no real need to remove it. With different configurations, it will get updated accordingly. It can speed up a little bit the first iOS install in RNTester.

If you prefer, I can keep this line and remove it.

Reply:
(also... it was commented... 😅)

Reply:
ok, let's keep your change then